src/components/strategies/marketPriceIndication/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import BigNumber from 'bignumber.js';
+import { Tooltip } from 'components/common/tooltip/Tooltip';
+
+type Props = {
+  marketPricePercentage: BigNumber;
+  isRange?: boolean;
```

Comment:
why is `isRange` optional? In what case do we pass undefined?

Reply:
MarketPriceIndication component is in use from 2 places only: 
InputLimit.tsx & inputRange.tsx so we know if we are in a isRange case or not. 
So, if you don't pass isRange prop, default would be false.